packages/web-components/scripts/addPreactAliases.js

```diff
@@ -0,0 +1,42 @@
+import { readFile, writeFile } from "node:fs/promises";
+import path from "node:path";
+import { createRequire } from "node:module";
+
+const require = createRequire(import.meta.url);
+
+const addPreactAliases = async () => {
+  // Determine the tsconfig.json location (assumed to be in the project root)
+  const tsconfigPath = path.join(process.cwd(), "tsconfig.json");
+  let tsconfig;
+
+  try {
+    const content = await readFile(tsconfigPath, "utf8");
+    tsconfig = JSON.parse(content);
+  } catch (err) {
+    process.exit(1);
```

Comment:
The wrapper does nothing because in case of error, the process fails with code 1